src/Compilers/CSharp/Portable/Declarations/DeclarationTable.cs

```diff
@@ -365,5 +366,8 @@ public IEnumerable<ReferenceDirective> ReferenceDirectives
 
             return false;
         }
+
+        public INamespaceDeclaration GetDeclarationRoot(CSharpCompilation compilation)
+            => GetMergedRoot(compilation);
```

Comment:
is this method actually needed?  couldn't this be inlined into the caller?

Reply:
Sure. Will inline